packages/core-data/src/fetch/__experimental-fetch-link-suggestions.js

```diff
@@ -153,6 +153,27 @@ const fetchLinkSuggestions = async (
 		);
 	}
 
+	if ( ! type || type === 'attachment' ) {
+		queries.push(
+			apiFetch( {
+				path: addQueryArgs( '/wp/v2/media', {
+					search,
+					page,
+					per_page: perPage,
+				} ),
+			} )
+				.then( ( results ) => {
+					return results.map( ( result ) => {
+						return {
+							...result,
+							meta: { kind: 'media' },
+						};
+					} );
+				} )
+				.catch( () => [] )
```

Comment:
Should we throw an error here?

Reply:
Probably needs looking into. The pattern is already in evidence so I'm following that for now.